.github/actions/test-linux/action.yml

```diff
@@ -16,7 +16,12 @@ runs:
         export PDO_MYSQL_TEST_PASS=root
         export PDO_DBLIB_TEST_DSN="dblib:host=127.0.0.1;dbname=master;version=7.0"
         export PDO_DBLIB_TEST_USER="pdo_test"
-        export PDO_DBLIB_TEST_PASS="password"
+        export PHP_OCI8_TEST_USER="system"
+        export PHP_OCI8_TEST_PASS="passXX"
```

Comment:
I tried to break the tests. Can someone please help me to understand why the oci8/pdo_oci tests are skipped but not listed as skipped? It seems they are excluded somewhere, but where?

Reply:
- Oracle Database now uses container and pluggable databases.  The XE service name is XEPDB1 so the connect string should be like "oracle:1521/XEPDB1", if "oracle" is the hostname of the DB container.
- Are OCI8 and PDO_OCI (and the Instant Client libraries) actually installed?

Reply:
yes, there are installed incl. the Client libs https://github.com/php/php-src/blob/c54ffa4c5d08442f177f626920e636b5ea860bb6/.github/actions/apt-x64/action.yml#L62

I wonder why the oci8/pdo_cci tests are not listed in SKIP lines in CI? @cmb69 any idea?

Reply:
Commonly LD_LIBRARY_PATH or ldconfig are needed.

Reply:
thank you, your comments helped me a lot